### ISSUE TITLE
Add Broadcast option in lichess app

### ISSUE THREAD

**Issue by LichessUser** [veloce/lichobile#1284]
Add an option to see live tournament broadcasts in the lichess app. It's a very useful feature, we need that in app. If anyone can make a pull request for it that would be great.
Thanks.

**Comment by lipis** [comment]
Yeah it would be great to follow the current tournaments like Tata Steel for example on the phone!
